assets/scss/bootstrap.scss

```diff
@@ -1,3 +1,9 @@
-div.bootstrap {
-    @import "~bootstrap";
+div.bootstrap, .tooltip {
```

Comment:
This means tooltip class gets bootstrap applied, is that really needed?

Reply:
The tooltip div spawns outside the parent div element by design, and none of the styles get applied to it